service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C21OrderEmailContentProvider.java

```diff
@@ -61,26 +57,15 @@ public C21OrderEmailContentProvider(@Value("${ccd.ui.base.url}")String uiBaseUrl
             .build();
     }
 
-    private String buildSubjectLineWithHearingBookingDateSuffix(final String subjectLine, final CaseData caseData) {
-        String hearingDate = "";
-        if (isNotEmpty(caseData.getHearingDetails())) {
-            hearingDate = " hearing " + dateFormatterService.formatLocalDateToString(
-                hearingBookingService.getMostUrgentHearingBooking(
-                    caseData.getHearingDetails()).getStartDate().toLocalDate(), FormatStyle.MEDIUM);
-        }
-        return Stream.of(subjectLine, hearingDate)
-            .filter(StringUtils::isNotBlank)
-            .collect(joining(","));
-    }
-
     private Map<String, Object> commonC21NotificationParameters(final CaseDetails caseDetails,
                                                                 final String linkToDocument) {
         CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
         final String subjectLine = buildSubjectLine(caseData);
         return ImmutableMap.of(
             "subjectLine", subjectLine,
             "linkToDocument", linkToDocument,
-            "hearingDetailsCallout", buildSubjectLineWithHearingBookingDateSuffix(subjectLine, caseData),
+            "hearingDetailsCallout", buildSubjectLineWithHearingBookingDateSuffix(subjectLine,
```

Comment:
I don't like this name `hearingDetailsCallout` as it describes where the property is to be positioned in a template rather than it's contents :/

Reply:
This is an existing method and I had to move _buildSubjectLineWithHearingBookingDateSuffix_  to make reusable. Updating the parameter will mean doing same for the C21 notification template. Do we want to do this?

Reply:
Yeah I thought that may of been the case, no let's leave it for now.